spec/check_localization_progress_spec.rb

```diff
@@ -0,0 +1,337 @@
+require 'spec_helper.rb'
+require 'webmock/rspec'
+
+describe Fastlane::Actions::CheckTranslationProgressAction do
+  before(:each) do
+    allow(FastlaneCore::UI).to receive(:'message')
+    allow(FastlaneCore::UI).to receive(:'interactive?').and_return(true)
+  end
```

Comment:
The behaviour is slightly different on CI and local, so this forces the tests to always run as in a local environment. This is the most likely scenario for this action and the logic on our side is pretty simple, so I opted for this rather than adding tests for all the cases which would basically end up testing Fastlane rather than our logic.